src/blog/2023/09/dashboard-notebook-layout.md

```diff
@@ -0,0 +1,88 @@
+
+<!--more-->
+
+In our v0.4.0 release, we've introduced a new "Notebook" layout, alongside a new Table widget.
+
+The Notebook layout is designed to allow users to create Dashboards structured like a Notebook (most often seen with the likes of [Jupyter Notebooks](https://jupyter.org/) or [ObservableHQ](https://observablehq.com/)).
+
+Here we will deepdive into the Notebook layout, and show how, alongside our new **Markdown Node** ([docs](https://flowforge.github.io/flowforge-nr-dashboard/nodes/widgets/ui-markdown.html)), **Table Node** ([docs](https://flowforge.github.io/flowforge-nr-dashboard/nodes/widgets/ui-table.html)) and others, it's becoming easier to create dynamic and interactive Dashboards.
+
+_Note: If you're not familiar with Markdown, it's a simple markup language that allows you to format text. You can learn more about it [here](https://www.markdownguide.org/cheat-sheet/)._
+
+## Dashboard Hierarchy
+
+As a quick introductory note ahead of our below guide, each Dashboard is structured accordingly:
+
+- **Widget**: An individual functional block, e.g. button, chart, slider
+- **Group**: A collection of widgets that render together
+- **Page**: A single page/tab in your Dashboard. Each page can have it's own Layout, in this case we'll use "Notebook"
+- **UI**: Contains a collection of pages, deployed from Node-RED, provides the basic side navigation to switch between Pages.
+
+## Building a Notebook
+
+![Example Notebook created in Dashboard](./images/db-notebook-example.png)
+
+To get started, drop your first widget (in this case, we'll add a `ui-markdown`) onto the Node-RED canvas. This in turn will prompt us to create our first Group/Page/Dashboard which we can name & configure accordingly.
+
+Let's add the following Markdown to our first widget:
+
+```md
+# Markdown Content
+
+Here we can render dynamic Markdown content that is
+easily _styled_.
+
+{% raw %}
+We can inject `msg.payload`. For example, here is a
+timestamp updating every second: {{ msg.payload }}
+{% endraw %}
+```
+
+The joy of `ui-markdown` in Dashboard 2.0 is _dynamic_ content, i.e. content that can be updated by passing messages to the `ui-markdown` node. We can wire an `inject` node, set it up to repeat every 1s, and connect it to `ui-markdown`. Now, our Markdown content will automatically update show this value.
+
+![Screenshot to show how an inject node can drive content of a ui-markdown node](./images/db-notebook-inject.png)
+
+Resulting in:
+
+![Dynamic markdown with an updating timestamp every 1 second](./images/md-timestamp.gif)
+
+## Adding More Widgets
+
+Because the Notebook is _just_ a layout, we can still wire together any of the available widgets and existing nodes and display them accordingly.
+
+Let's wire a ui-button, HTTP Request and ui-table node. When we click the button, it will perform the HTTP request, and then render the response in the table.
```

Comment:
```suggestion
Let's wire a `ui-button`, HTTP Request, and `ui-table` node. When we click the button, it will perform the HTTP request, and then render the response in the table.
```